src/plugins/discover/public/application/angular/directives/histogram.tsx

```diff
@@ -298,10 +297,10 @@ export class DiscoverHistogram extends Component<DiscoverHistogramProps, Discove
     }
 
     const rectAnnotationStyle = {
-      stroke: isDarkMode ? darkEuiTheme.euiColorLightShade : lightEuiTheme.euiColorDarkShade,
+      stroke: isDarkMode ? euiThemeVars.euiColorLightShade : euiThemeVars.euiColorDarkShade,
```

Comment:
Not sure where this is used, but I know of other cases where we just used one or the other no matter dark or light theme, as shades flip in one to the other. Just commenting to make sure that was accounted for

Reply:
Yeah, I'm not convinced of this pattern, either, but I decided it was best to keep the existing behavior/definition. The change just ensures that it gets the values from the correct theme.